### PR TITLE
Use latest version of flutter actions

### PR DESCRIPTION
**Pull request by Persson-dev**

### CODE REVIEW

**Review by hatch01 — APPROVED** [review]

**Comment by Persson-dev** [comment]
Well, Android is still failing (same error as before)

**Comment by hatch01** [comment]
> Well, Android is still failing

maybe a java version problem

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -11,11 +11,11 @@ jobs:
    runs-on: ubuntu-latest
    steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-java@v2
+    - uses: actions/setup-java@v3
       with:
         distribution: 'zulu'
         java-version: '11'
```
Comment (uku3lig) [inline .github/workflows/android.yml]: you're building with java 11 (class file v55.0) when java 17 seems to be needed (class file v61.0)

**Comment by flashonfire** [comment]
It's clearly a java version problem as pointed out by @uku3lig, I was able to build it locally with java 17.

**Review by uku3lig — APPROVED** [review]

**Pull request merged**